### DISCUSSION TITLE
Retrieve the ID of each product woocommerce

### DISCUSSION THREAD

**Discussion by Vichrist12** [woocommerce/woocommerce#34882] [WooCommerce HPOS]
Hi everyone I am a junior web developer and I would like to display the ID of each product so that the customer can make a purchase with it can I do it how?

**Comment by shaybanshee** [comment]
Hi @Vichrist12 this is specifically a discussion forum for issues related to the High Performance Order Storage update. For more general questions and guidance please sign up for our [Community Slack](https://woocommerce.com/community-slack/) and post in the appropriate channels. [Stack Overflow ](https://stackoverflow.com/questions/tagged/woocommerce)also has many answers to questions such as this!
